core/src/main/java/com/opensymphony/xwork2/interceptor/ParametersInterceptor.java

```diff
@@ -101,7 +104,7 @@ static private int countOGNLCharacters(String s) {
     static final Comparator<String> rbCollator = new Comparator<String>() {
         public int compare(String s1, String s2) {
             int l1 = countOGNLCharacters(s1),
-                l2 = countOGNLCharacters(s2);
+                    l2 = countOGNLCharacters(s2);
```

Comment:
```suggestion
            int l1 = countOGNLCharacters(s1);
            int l2 = countOGNLCharacters(s2);
```

Maybe it's better (more clear) to split this into two separate assignments?

Reply:
👍